CMP1903_A1_2324/Testing.cs

```diff
@@ -1,20 +1,25 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
-namespace CMP1903_A1_2324
-{
-    internal class Testing
-    {
-        /*
-         * This class should test the Game and the Die class.
-         * Create a Game object, call the methods and compare their output to expected output.
-         * Create a Die object and call its method.
-         * Use debug.assert() to make the comparisons and tests.
-         */
+namespace CMP1903_A1_2324 {
+  internal class Testing {
 
-        //Method
+  //Method
+    public void Test() {
+      Game game = new Game(); //Create new game object
+      List<int> tempList = game.RollDie(); //Make a temp list and assign its value to the return of the game object RollDie method
+      int sum = tempList.Sum(); //Make an integer variable, assign its value to be the sum of the items in the list
+
+      Debug.Assert(sum < 19 && sum > 2, "Sum is outside expected range"); //Check to ensure the sum is in the range 3 <= x <= 18
+    
+      //Check to ensure all rolls are within the valid range
+      for (int i = 0; i < tempList.Count; i++) {
+        Debug.Assert(tempList[i] > 0 && tempList[i] < 7, "Roll is outside of expected range");
+      }
```

Comment:
All elements are added nicely but i have noticed for the testing class it is asked to create a new Die object and call its methods using the die object instead of using the game class. However you are still testing both classes correctly either way

Formatting is consistent again to the style guide

commenting allows for very readable code